### PR TITLE
Drop support for Flash

### PR DESCRIPTION
**Pull request by R4356th**
Drops support for Flash since it is officially dead now per discussion on Discord.

### CODE REVIEW

File: YouTube.class.php [hunk 1]
```diff
@@ -97,44 +97,15 @@ public static function embedYouTube( $input, $argv, $parser ) {
...
-			$maxWidth = 960;
-			$maxHeight = 720;
-
-			if (
-				!empty( $argv['width'] ) &&
-				filter_var( $argv['width'], FILTER_VALIDATE_INT, [ 'options' => [ 'min_range' => 0 ] ] ) &&
+		if (
+			!empty( $argv['width'] ) &&
+			filter_var( $argv['width'], FILTER_VALIDATE_INT, [ 'options' => [ 'min_range' => 0 ] ] ) &&
 				$argv['width'] <= $maxWidth
```
Comment (The-Voidwalker) [inline YouTube.class.php:109]: There's some weird indentation stuff going on here, you should probably remove an indent from every line following this one until the end of the function.

**Review by The-Voidwalker — CHANGES_REQUESTED** [review]

**Review by Universal-Omega — COMMENTED** [review]
Looks good to me besides what Void mentioned. I'll merge once that's fixed.

**Comment by Universal-Omega** [comment]
> Looks good to me besides what Void mentioned. I'll merge once that's fixed.

On second thought I'm not sure I see the issue?

File: YouTube.class.php (hunk 1 above)
Comment (Universal-Omega) [reply YouTube.class.php:109]: I don't really see the issue when looking at it, it looks clean/indented correctly.

Comment (R4356th) [reply YouTube.class.php:109]: This is how it was originally indented.

Comment (Universal-Omega) [reply YouTube.class.php:109]: Yeah that's what I figured. Which is why I merged.

**Pull request merged**